packages/auth/src/core/index.ts

```diff
@@ -66,32 +65,24 @@ export function setPersistence(
  * Changes the reCAPTCHA configuration on the `Auth` instance.
  *
  * @remarks
- * This will apply the reCAPTCHA config to the current Auth session and affect future auth
- * requests.
+ * This will pull the reCAPTCHA config to the current Auth session and affect future auth
+ * requests, which indicates whether the reCAPTCHA verification flow should be triggered for a
+ * specific auth provider. If initializeRecaptchaConfig() is not invoked, the auth flow will start
+ * without reCAPTCHA verification. But if reCAPTCHA verificaiton is required, the reCAPTCHA config
+ * will be automatically pulled internally and the flows will restart. Thus calling this method
```

Comment:
Maybe the last line can be something like: "Thus, calling this optional method will reduce the latency of auth flows."